lib/templates/immutable.yaml.mustache

```diff
@@ -19,6 +19,10 @@
       region: '{{=<% %>=}}{{ region }}<%={{ }}=%>'
       aws_zone: '{{=<% %>=}}{{ region }}<%={{ }}=%>{{=<% %>=}}{{ aws_zone }}<%={{ }}=%>'
       group: '{{{ security_group }}}'
+      {{#using_vpc}}
+      vpc_subnet_id: '{{{ vpc_subnet_id }}}'
+      assign_public_ip: yes
```

Comment:
I think we probably need this regardless of whether it's in classic or not? I'm not sure